src/components/TodoInfo/TodoInfo.scss

```diff
@@ -0,0 +1,17 @@
+.TodoInfo {
+  padding: 8px;
+  margin: 12px 0;
+  border: 1px solid #000;
+  border-radius: 8px;
+  background-color: antiquewhite;
+
+  &__title {
```

Comment:
Consider using BEM naming convention for better readability and maintainability. 🤓